packages/components/src/components/icon/icon.tsx

```diff
@@ -89,6 +89,15 @@ export class Icon {
     );
 
     // TODO: wouldn't it be better to set alt attribute instead of aria-label?
-    return <img src={buildIconUrl(this.source || this.name)} {...parseAndGetAriaAttributes(this.aria)} alt="" />;
+    return (
+      <img
+        src={buildIconUrl(this.source || this.name)}
+        {...parseAndGetAriaAttributes(this.aria)}
+        width={24} // improve bootstrapping behaviour
```

Comment:
Is this valid for custom icons passed by `source` prop too?

Reply:
As discussed in private chat, we should add the attributes also for custom sources